.github/workflows/cd_support_sphere_py.yml

```diff
@@ -93,3 +95,39 @@ jobs:
 
       - uses: pypa/gh-action-pypi-publish@v1.9.0
         if: startsWith(github.ref, 'refs/tags')
+
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    needs: [test-built-dist]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delay for PyPI
+        run: sleep 5 # Wait for the package to be available in PyPI
+      - name: Check out the repo
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+      - name: Log in to registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v4
+        with:
+          context: ./deployment/docker/
+          file: ./deployment/docker/populate_db.Dockerfile
+          platforms: linux/amd64
```

Comment:
any reason not to build for `linux/arm64` as well? don't think we're using it right now so it's probably fine to omit -- only asking bc this differs from the NoisePy setup

Reply:
@nikiburggraf - I have added Linux/arm64 as well. But just to confirm, can you please let me know the NoisePy setup file you are looking at? I checked this one (only having Linux/amd64):
https://github.com/noisepy/NoisePy/blob/main/.github/workflows/release.yaml#L131C22-L131C33


Proof of testing: https://github.com/anujsinha3/post-disaster-comms/actions/runs/11245663655/job/31266137232